File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        
        s = s.split(' ')
        print(s)
        
        res = []
        for x in reversed(range(len(s))):
            if s[x]:
                res.append(s[x])
                
        return ' '.join(res)
        

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    s = s.split(' ')
    let res = []
    for(let i = s.length; i >= 0; i--){
        if(s[i]){
            res.push(s[i])
        }
    }
    return res.join(' ')
};